src/api/app/controllers/webui/users/notifications_controller.rb

```diff
@@ -21,13 +21,14 @@ def update
                       fetch_notifications.where(id: params[:notification_ids])
                     end
     # rubocop:disable Rails/SkipsModelValidations
-    updated_notifications = notifications.update_all('delivered = !delivered')
+    number_of_marked_as_read = notifications.where(delivered: false).update_all('delivered = !delivered')
```

Comment:
What do you think of `read_count`? Is smaller, but do you think it conveys the same meaning?

Reply:
I think it can work